### PR TITLE
Pip interoperability

### PR DESCRIPTION
**Pull request by rrigdon**

### CODE REVIEW

File: docs/source/user-guide/configuration/pip-interoperability.rst [hunk 1]
```diff
@@ -0,0 +1,33 @@
+===================================
+Improving interoperability with pip
+===================================
+
+To improve the interoperability between conda and pip,
+conda 4.6.0 added preview support. With this interoperability,
```
Comment (jjhelmus) [inline docs/source/user-guide/configuration/pip-interoperability.rst]: `preview support` sounds awkward.  Perhaps:

The conda 4.6.0 release added improved support for interoperability between conda and pip.  This feature is still experimental and is therefore off by default.

File: docs/source/user-guide/configuration/pip-interoperability.rst [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+conda can use pip-installed packages to satisfy dependencies,
+cleanly remove pip-installed software, and replace them with
+conda packages when appropriate.
+
+If you’d like to try the feature, you can set this ``.condarc`` setting::
+
+   conda config --set pip_interop_enabled True
+
```
Comment (jjhelmus) [inline docs/source/user-guide/configuration/pip-interoperability.rst]: A note informing users that setting `pip_interop_enabled` to True may slow down conda would be a good addition here.

File: docs/source/user-guide/configuration/index.rst [hunk 3]
```diff
@@ -7,7 +7,9 @@ Configuration
 
    use-condarc
    sample-condarc
+   free-channel
```
Comment (jjhelmus) [inline docs/source/user-guide/configuration/index.rst]: I do not see this file in this PR.  Was it not added to the git repo?

Comment (rrigdon) [reply docs/source/user-guide/configuration/index.rst]: It's in a different PR that's supposed to go in today. I'll delete the reference here and fix it when the next PR goes through.

**Pull request merged**